### PR TITLE
handle load tests assetion for jmeter

### PR DESCRIPTION
**Pull request by SergeyMoroz0703**

### CODE REVIEW

File: app/jmeter/jira.jmx [hunk 1]
```diff
@@ -3061,12 +3061,12 @@ if ( sleep_time &gt; 0 ) {
                 <hashTree/>
                 <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
                   <collectionProp name="Asserion.test_strings">
-                    <stringProp name="1182867844">[&quot;project-key&quot;]=&quot;\&quot;${project_key}\&quot;&quot;;</stringProp>
+                    <stringProp name="-1259658522">.*project-key.*${project_key}</stringProp>
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:3064]: What is the name here? Would it be possible to have a meaningful name here or is this an autogenerated thing?

File: app/jmeter/jira.jmx [hunk 2]
```diff
@@ -3061,12 +3061,12 @@ if ( sleep_time &gt; 0 ) {
...
                   </collectionProp>
                   <stringProp name="Assertion.custom_message"></stringProp>
                   <stringProp name="Assertion.test_field">Assertion.response_data</stringProp>
                   <boolProp name="Assertion.assume_success">false</boolProp>
-                  <intProp name="Assertion.test_type">16</intProp>
+                  <intProp name="Assertion.test_type">2</intProp>
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:3069]: What is this 16 and 2?

File: app/jmeter/jira.jmx [hunk 3]
```diff
@@ -9602,7 +9602,7 @@ if ( sleep_time &gt; 0 ) {
           <hashTree/>
         </hashTree>
       </hashTree>
-      <ResultCollector guiclass="ViewResultsFullVisualizer" testclass="ResultCollector" testname="View Results Tree" enabled="false">
+      <ResultCollector guiclass="ViewResultsFullVisualizer" testclass="ResultCollector" testname="View Results Tree" enabled="true">
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:9605]: Need to disable View Results Tree element before merge.

File: app/jmeter/jira.jmx (hunk 2 above)
Comment (SergeyMoroz0703) [reply app/jmeter/jira.jmx:3069]: It is type of "matching" rules.
Like 2 - "Contains"
16 - "Substring", etc.

File: app/jmeter/jira.jmx (hunk 1 above)
Comment (SergeyMoroz0703) [reply app/jmeter/jira.jmx:3064]: It is auto-generated name for pattern in assetion

**Review by ometelytsia — APPROVED** [review]

**Pull request merged**